lib/formdata/0.0.2/index.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * FormData polyfill for k6
+ * Copyright (C) 2021 Load Impact
+ * License: MIT
+ *
+ * This simplifies the creation of multipart/form-data requests from k6 scripts.
+ * It was adapted from the original version by Rob Wu[1] to remove references of
+ * XMLHttpRequest and File related code which isn't supported in k6.
+ *
+ * [1]: https://gist.github.com/Rob--W/8b5adedd84c0d36aba64
+ **/
+
+if (exports.FormData) {
+  // Don't replace FormData if it already exists
+  return;
+}
+
+// Export variable to the global scope
+exports.FormData = FormData;
+
+function FormData() {
+  // Force a Constructor
+  if (!(this instanceof FormData)) return new FormData();
+  // Generate a random boundary - This must be unique with respect to the
+  // form's contents.
+  this.boundary = '------RWWorkerFormDataBoundary' + Math.random().toString(36);
+  this.parts = [];
+
+  /**
+   * Internal method. Convert input to a byte array.
+   * @param inp String | ArrayBuffer | Uint8Array  Input
+   */
+  this.__toByteArray = function(inp) {
+    var arr = [];
+    var i = 0, len;
+    if (typeof inp == 'string') {
+      for (len = inp.length; i < len; ++i)
+        arr.push(inp.charCodeAt(i) & 0xff);
+    } else if (inp && inp.byteLength) {/*If ArrayBuffer or typed array */
+      if (!('byteOffset' in inp))   /* If ArrayBuffer, wrap in view */
+        inp = new Uint8Array(inp);
+      for (len = inp.byteLength; i < len; ++i)
+        arr.push(inp[i] & 0xff);
+    }
+    return arr;
+  };
+}
+
+/**
+ * @param fieldName          String                    Form field name
+ * @param data               object|string             An object or string field value.
+ *
+ * If data is an object, it should match the structure of k6's http.FileData
+ * object (returned by http.file()) and consist of:
+ * @param data.data          String|Array|ArrayBuffer  File data
+ * @param data.filename      String                    Optional file name
+ * @param data.content_type  String                    Optional content type, default is application/octet-stream
+ **/
+FormData.prototype.append = function(fieldName, data) {
+  if (arguments.length < 2) {
+    throw new SyntaxError('Not enough arguments');
+  }
+  var file = data;
+  if (typeof data == 'string') {
```

Comment:
Is there any reason why we wouldn't want to do strict equality here?

Reply:
Personal habit :) `typeof data` wouldn't return anything other than a string, so a strict equality check doesn't seem necessary. But sure, I'll change it.